addons/website_sale/static/src/js/website_sale_options.js

```diff
@@ -131,7 +131,7 @@ sAnimations.registry.WebsiteSaleOptions = sAnimations.Class.extend(ProductConfig
      */
     _onOptionsUpdateQuantity: function (quantity) {
         this.$form
-            .find('input[name="add_qty"]')
+            .find('.js_main_product input[name="add_qty"]')
```

Comment:
Missing opw- in the commit message

Reply:
Well this commit is not for the opw, it's just a bug I found while testing my fix... Do I add the opw in that case?

Reply:
Boarf, if the PR is mentioned, ok.